outbox/docker-compose.yaml

```diff
@@ -55,23 +64,39 @@ services:
     ports:
       - "${PGADMIN_PORT:-5050}:80"
   connect:
-    image: debezium/connect:${DEBEZIUM_VERSION}
+    image: strimzi/kafka:0.20.0-kafka-2.6.0
     ports:
      - 8083:8083
-     - 5005:5005
     links:
      - kafka
      - order-db
     environment:
-     - BOOTSTRAP_SERVERS=kafka:9092
-     - GROUP_ID=1
-     - CONFIG_STORAGE_TOPIC=my_source_connect_configs
-     - OFFSET_STORAGE_TOPIC=my_source_connect_offsets
-     - STATUS_STORAGE_TOPIC=my_source_connect_statuses
-     - CONNECT_KEY_CONVERTER_SCHEMAS_ENABLE=false
-     - CONNECT_VALUE_CONVERTER_SCHEMAS_ENABLE=false
+     - KAFKA_LOG4J_OPTS=-Dlog4j.configuration=file:/opt/kafka/config/connect-log4j.properties
+     - KAFKA_CONNECT_BOOTSTRAP_SERVERS=kafka:9092
+     - |
+         KAFKA_CONNECT_CONFIGURATION=
+         key.converter=org.apache.kafka.connect.json.JsonConverter
+         value.converter=org.apache.kafka.connect.json.JsonConverter
+         key.converter.schemas.enable=false
+         value.converter.schemas.enable=false
+         group.id=connect
+         offset.storage.topic=connect-offsets
+         offset.storage.replication.factor=1
+         config.storage.topic=connect-configs
+         config.storage.replication.factor=1
+         status.storage.topic=connect-status
+         status.storage.replication.factor=1
+         consumer.interceptor.classes=io.opentracing.contrib.kafka.TracingConsumerInterceptor
+         producer.interceptor.classes=io.opentracing.contrib.kafka.TracingProducerInterceptor
+     - JAEGER_SERVICE_NAME=kafka-connect
+     - JAEGER_AGENT_HOST=jaeger
+     - STRIMZI_TRACING=jaeger
      - KAFKA_DEBUG=true
      - DEBUG_SUSPEND_FLAG=n
+     - JAVA_DEBUG_PORT=*:5005
+    volumes:
+      - $HOME/tmp/debezium:/opt/kafka/plugins
```

Comment:
Can we find a more portable way for this one :)

Reply:
See the comment, this is workaround till at least snapshot with the function is out. Then the curl will be used to build custom image. Till this is done the PR will stay as draft